src/utils.tsx

```diff
@@ -61,3 +65,19 @@ export const getPathFromQueryString = (history: History) => {
 
   return typeof path === 'string' && path.length ? path : null;
 };
+
+type GetCodeLineAnchorGetterParams = {
+  _getCodeLineAnchor?: typeof getCodeLineAnchor;
+  compareInfo?: CompareInfo | null | void;
+};
+
+export const getCodeLineAnchorGetter = ({
```

Comment:
I suggest calling this `createCodeLineAnchorGetter()` so that, at a glance, you could immediately see it's different from `getCodeLineAnchor()`